Tests/SonarQube.TeamBuild.PreProcessor.Tests/DataModel/ServerDataModel.cs

```diff
@@ -101,5 +109,37 @@ private static string GetEmbeddedFileKey(string pluginKey, string embeddedFileNa
             return pluginKey + "___" + embeddedFileName;
         }
         #endregion
+
+
+        #region Private methods
+
+        private byte[] CreateDummyZipFile(params string[] contentFileNames)
+        {
+            string fileName = "dummy.zip";
+
+            // Create a temporary directory structure
+            string tempDir = Path.Combine(Path.GetTempPath(), "sqTestsTemp", System.Guid.NewGuid().ToString());
```

Comment:
This is the only material change. Previously the temp directory was being created under the current test directory. However, the _TestContext_ isn't available so it's being created under %temp% instead. The temp directory is deleted at the end of the method anyway, so it doesn't make any difference.